fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -659,9 +676,13 @@ struct LogGenerationData : NonCopyable, public ReferenceCounted<LogGenerationDat
 
 	// only callable after getStorageTeamData returns a null reference
 	Reference<StorageTeamData> createStorageTeamData(StorageTeamID team,
-	                                                 std::vector<Tag>& tags,
-	                                                 std::map<Tag, Version> tagToVersions = {}) {
-		return storageTeamData[team] = makeReference<StorageTeamData>(team, tags, tagToVersions);
+	                                                 const std::vector<Tag>& tags,
+	                                                 bool nothingPersistent,
+	                                                 bool poppedRecently,
+	                                                 bool unpoppedRecovered,
```

Comment:
Nit: consider using `FDB_BOOLEAN_PARAM` for better readability and type checking.